src/main/webapp/utilities.js

```diff
@@ -0,0 +1,78 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// A series of utility functions that will be helpful throughout the front-end
+
+/* eslint-disable no-unused-vars */
```

Comment:
Was ESLint complaining about unused funcs?

Reply:
Yes, due to the `deleteCookie` function which is used in later, dependent PRs.